data/ledger.go

```diff
@@ -43,6 +44,23 @@ type Ledger struct {
 	*ledger.Ledger
 
 	log logging.Logger
+
+	lastRoundCirculation atomic.Value
+	lastRoundSeed        atomic.Value
+}
+
+type roundCirculation struct {
+	prevRound       basics.Round
+	prevOnlineMoney basics.MicroAlgos
+	curRound        basics.Round
+	curOnlineMoney  basics.MicroAlgos
+}
+
+type roundSeed struct {
+	prevRound basics.Round
```

Comment:
prevRound indicates the previously stored round, not the most previous round right? Might be helpful to include a comment clarifying this / rename the field

Reply:
I'm not sure what `most previous round` would be. It's a two element moving window cache.
The implementation here is the trivial one, allowing to reduce the pressure from the accounts update.
My intent is eventually to wrap this into the agreement itself, and delete this code completely.
( when the agreement is making ledger advancements, it should query these values and store these locally.
we should go "outside" the agreement "box" only when needed )

Reply:
right so `prevRound` could be confused with `curRound - 1`. But this is not the case here, so I suggested a comment possibly clarifying.

Reply:
I've added some documentation which I hope would be clear enough.

Reply:
I'm not a big fan of a/b; how do you feel about an array of two elements ?

Reply:
2 element array sounds great

Reply:
ok, done.